exe/RadicleExe.hs

```diff
@@ -45,6 +49,13 @@ readSource file = do
        then readFile file
        else die $ "Could not find file: " <> toS file
 
+-- | Since `#` is not a comment in radicle, we need to explictly remove shebang
+-- lines.
+ignoreShebang :: Text -> Text
+ignoreShebang src = case T.lines src of
```

Comment:
I think the shebang should be handled gracefully in other cases as well: When reading from stdin, when using `load!` or `read-file!` or `file-module`.

Reply:
It feels to me like `read-file!` should definitely not have it - the file may well not even be radicle code, or code at all. But I can add it to `load!` and `file-module!`.